### ISSUE TITLE
Error: misaligned address

### ISSUE THREAD

**Issue by MisterTeo** [brichard19/BitCrack#268]
After a "forced" restart, I get "error: misaligned address". I reinstalled windows, visual studio, cuda...all drivers...error persist. I tried to change -t -b and -p value...still persist. I tried with standard comand...with just 1 bitcoin address...still "error: misaligned address".
Anyone with a solution. I have nothing more...

**Comment by maxya** [comment]
Same here after latest NVIDIA driver update, cuBitCrack.exe stopped working with "error: misaligned address"

Changed to clBitCrack.exe and its working just fine with same config, just less hashes..

**Comment by PsyShark** [comment]
> After a "forced" restart, I get "error: misaligned address". I reinstalled windows, visual studio, cuda...all drivers...error persist. I tried to change -t -b and -p value...still persist. I tried with standard comand...with just 1 bitcoin address...still "error: misaligned address".
> Anyone with a solution. I have nothing more...

Go back to the previous driver (452.06)

**Comment by MisterTeo** [comment]
I switched to linux for now with version of djarumlights with random and all pull requests added...works ok, little faster than original...

**Comment by afzznana** [comment]
> 
> 
> > After a "forced" restart, I get "error: misaligned address". I reinstalled windows, visual studio, cuda...all drivers...error persist. I tried to change -t -b and -p value...still persist. I tried with standard comand...with just 1 bitcoin address...still "error: misaligned address".
> > Anyone with a solution. I have nothing more...
> 
> Go back to the previous driver (452.06)

After installed driver (452.06). it's still get "error: misaligned address". Any solution. I have nothing more... TQ
![ERROR - misaligned address - bitcrack](https://user-images.githubusercontent.com/72477683/95273857-34711280-0855-11eb-9d3e-9a1fc5556030.jpg)

**Comment by mirt23** [comment]
Same issue here and there is no way to make it work again. Cl works fine. Any new ideas how to fix it?

**Comment by MisterTeo** [comment]
Worked for me with old driver. Try it.

**Comment by mirt23** [comment]
Thanks, Ill try a couple of things as the new drivers are useful as I use it as a workstation during the day and bitcrack at night.

**Comment by Gman54** [comment]
Any way this can be addressed? I'm being forced to update drivers for my RTX 2080 to work with other programs,

**Comment by ZenulAbidin** [comment]
To everyone having this problem, see https://bitcointalk.org/index.php?topic=4453897.msg56053911#msg56053911 for a description of where it's coming from.

TL;DR it can be patched, but it requires adjustments to ether the secp256k1 or cudaKeySearchDevice CUDA code.
